### DISCUSSION TITLE
Question! What are you using the webUI for?

### DISCUSSION THREAD

**Discussion by monkeymademe** [monkeymademe/CamUI#67] [General]
Just a question to all the webUI users. What are you using it for?

**Comment by dalogue1** [comment]
Video and photos of my birdfeeder.

**Comment by AleksanderKruecken** [comment]
Running it on RPI Zero 2 on OS Raspberry Bookworm desktop and showing live video in Home assistant.
I'm looking if I can also include it in frigate NVR...

**Reply by RickRoz** [reply]
How did u get the live video in home assistant?

**Comment by davidhorsell** [comment]
Running this on a Pi Zero 2 with Raspberry Pi OS Lite and the Pi Global Shutter camera. Works well so far and no bugs to report. (With the Lite OS, I needed to install git, python3-flask, python3-pillow and python3-picamera2 via 'apt' and then everything worked fine.)

I currently use RPi-Cam-Web-Interface on a Pi Zero 1 in undergraduate science laboratories. I am hoping to move to Pi Zero 2 but without having to resort to enabling legacy camera support. I am testing out picamera2-WebUI now as it looks more suitable for what the undergraduates need than RPi-Cam-Web-Interface.

For picamera2-WebUI to be a replacement, I would need buttons for start/stop of timelapse (which looks like it is planned) and also start/stop video recording. Is the latter possible? I tried looking at the code myself but could not see how to tinker with it without better understanding of the web page end of things.

**Reply by monkeymademe** [reply]
Timelapse is next for sure ;-)

**Comment by monkeymademe** [comment]
I would love to see some results of what you are doing ;-)

**Comment by codemonkey2k5** [comment]
I have it connected to my Prusa XL 3D printer to monitor print jobs.

**Comment by galoki** [comment]
I use it to watch my 3d printer.  Used to use octoprint with a webcam but with Prusa Connect I've abandoned octoprint and CamUI is a perfect replacement for the video feed.  I run it on a Pi5.

**Reply by monkeymademe** [reply]
Do you connect the feed to Prusa Connect or just watch in CamUI?

**Reply by galoki** [reply]
CamUI.  Prusa Connect wants a webcam or an API.

**Comment by benlamnz** [comment]
Along with 2 RPI 5 and 2 RPI Camera 3 HQ, this is my go-to daily camera setup. Nowadays only ever use my Samsung S23 for outdoor or ad-hoc photography. My only humble wish is for the exposure settings to be alterable via direct numeric entry - currently using the direction key to dial it up or down per millisecond is very time-consuming. If you use the slider, it always overshoots big time. An alternative suggestion is for the controls to be mappable to analogue joysticks, or to a DAW interface frequently seen with MIDI-keyboards.  

But that does not in any way detract from CAM-UI being the killer app, turnkey solution for the Pi-Camera HQ. This should be upstreamed to PI-OS as the default software; it is on par with GIMP, OBS, VLC, etc in terms of necessity to a complete PI-OS photography experience. Well done and thank you.

**Comment by AlfonsNiederhofer** [comment]
I want to use it in an educational context of astronomy. the wide range of exposure time is very interesting for spectroscopy and astrophotography.

**Comment by Bluesguy1984** [comment]
I want to use this in conjunction with motioneye so I can monitor multiple cameras at the same time and be able to save the video output to review at a later time.
